happi/tests/test_device.py

```diff
@@ -5,23 +5,22 @@
 
```

Comment:
Can we rename this file? test_device -> test_item

Reply:
[0f70bf4](https://github.com/pcdshub/happi/pull/266/commits/0f70bf41e7e8c62f1a23db23b4b523d253f18ca6)